### ISSUE TITLE
Marketing: Review Sites - G2

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#53]
We've experimented with review sites before and had some issues, but this idea came up again in the Marketing brainstorm so I've reached out to see how/if we can move things forward. 

The reason for working with G2 and other, comparable platforms is...

- Big SEO presence, even for general terms
- Enables leads to directly compare PostHog to other platforms
- Creates social proofs we can use in other marketing materials
- Ability to enter awards, magic quadrants, etc.
- Social proofs are effective for word of mouth

At higher levels of investment we can also start to leverage G2 as a lead-gen platform, integrate with CRM, etc. But we aren't there yet. 

### Previously...
In the past, we created [a free profile on G2](https://www.g2.com/products/posthog/reviews). I was in conversation with G2 about turning this into a paid profile, but we were unsure if this would be effective space for developer audiences or if we'd be best off driving reviews ourselves. 

As a result, [I manually reached out to all our YC deal companies and asked them for a review](https://github.com/PostHog/company-internal/issues/356). I contacted approx 40 companies. We got one review, from Pry. G2 were not _great_ to communicate with and we didn't have any demographic from them which gave us confidence in it as a dev-focused channel.

Eventually we realised that if we were to proceed, we'd need to invest in a paid plan. G2 would then give us a better profile and, if we provided the email addresses of users we wanted to target, would help us invite users to leave a review. However, these invites would clearly be from G2. We felt at the time that this was a step against our users, so we broke the conversation off. 

### Currently...
I've been speaking with G2 again recently, to get latest pricing and address previous concerns. Here's there pricing. We definitely don't need anything more than the Essential package, at this stage. 

![unnamed (1)](https://user-images.githubusercontent.com/84011561/170019427-90d0294f-7b70-4def-b90a-443f2b8f6297.png)

In terms of traffic and SEO value, here's a comparison of how our _current_ profile compares to [FullStory's paid profile](https://www.g2.com/products/fullstory/reviews?__cf_chl_tk=1RZa2YW1H.kkgy3T_NMfuSuzMxr_cQK2aKmUa4TJaxk-1653390138-0-gaNycGzNDf0). 

<img width="1110" alt="c5d5715b-cd60-4d61-93b9-63452e16a3ff" src="https://user-images.githubusercontent.com/84011561/170019668-7b40e38e-9fec-43a4-b0ed-09a5cb34032d.png">

Here's also [a case study of a somewhat similarly technical product](https://learn.g2.com/appknox-increases-product-pageviews), which saw positive results after investing in G2 as a channel. 

I also have some demographic data by industry, which I'm waiting on. However, due to G2's massive presence on search it's likely that the channel will still be seen by our audience even if engineers are not broadly aware of G2 as a trusted platform. 

### Next...

I'm speaking with G2 again on Thursday and will be trying to get a better understanding of how the review process would work and how we may be able to work with G2 without betraying any user privacy or handing over data to be stored/used by them. 

In the mean time, I'd love to hear any feedback on this idea and if we think it's still worth/not worth going ahead with. @charlescook-ph @andyvan-ph @simfish85 @camerondeleone @fuziontech

**Comment by andyvan-ph** [comment]
Overall, still pretty positive on this idea. I think the social proof point is pretty vital. I think it's an important way for us to be seen next to established names, and the cost is acceptable in comparison to the cost of paid ads etc.

**Comment by charlescook-ph** [comment]
Yep, agree we should still go ahead with this, thanks @joethreepwood ! Couple of thoughts to make this successful:

- I'm assuming we put all our eggs in one G2 basket and make that really successful (rather than covering multiple review sites)
- Just confirming you're happy to be the owner for this and responsible for keeping it updated etc.? (I'm also happy to help, just don't want it to fall through the cracks as a shared-responsibility type thing)

We _definitely_ can't go ahead in any scenario which requires us to share user data, but it hopefully sounds like this is one of those optional 'value adds' which we don't actually need to make use of...

**Comment by joethreepwood** [comment]
[Previously mentioned demographic info.](https://docs.google.com/presentation/d/1hNSNL_zBCj-Vw4tALv3V-2_fzKg2vhPx-Osv0j8Pyrw/edit?usp=sharing) Short version: It's mostly mid-level employees in mid-sized tech businesses. No view on department.

G2 is definitely the best place to start and we should absolutely put our eggs there _unless_ we have an idea of an alternative which is more focused on engineers. I've asked and researched, but not come up with any good alternatives. 

Absolutely happy to own it.

**Comment by joethreepwood** [comment]
Chatted this through with G2 yesterday. 

We can set up a workflow where we're not sending them data or sharing messages in their branding. However to do this, I'll have to run the campaigns manually or else set up some sort of automation to invite reviews.

They've also set up a new system since last time which enables us to reward users by making charitable donations instead of sending giftcards. Charities have to vetted, but we could submit DjangoGirls, Lesbians Who Tech, etc. Girls Who Code is already on their vetted list. 

If we go ahead, I'll probably run it manually and batch review requests based on active users for a few weeks, potentially splitting rewards into a A/B test. That'll let us vet how valuable the activity is and gauge how much work it is/if it's worth automating or bring into our user flows. 

Unless there are other concerns, we should be set to make a decision. They've sent me a contract which I'll forward over to you @charlescook-ph -- unless you're happy for me to sign?

**Comment by joethreepwood** [comment]
Oh, and at G2's end they'd be making landing pages like [this](https://www.g2.com/g2gives/one-tree-planted-pillar-2021/and/vvenues). This is what our email invites would forward users to.

**Comment by charlescook-ph** [comment]
Yep, let's go ahead (I read the order form, happy for you to sign)

**Comment by joethreepwood** [comment]
Contract signed. US is off today, but will update with next steps soon.

**Comment by joethreepwood** [comment]
will close this for now and create a new issue if needed
